plugins/integrations/kubernetes-service/src/main/java/com/cloud/kubernetes/cluster/KubernetesClusterManagerImpl.java

```diff
@@ -1329,6 +1479,82 @@ public boolean upgradeKubernetesCluster(UpgradeKubernetesClusterCmd cmd) throws
         return upgradeWorker.upgradeCluster();
     }
 
+    @Override
+    public boolean addVmsToCluster(AddVmsToKubernetesClusterCmd cmd) {
+        if (!KubernetesServiceEnabled.value()) {
+            logAndThrow(Level.ERROR, "Kubernetes Service plugin is disabled");
+        }
+        List<Long> vmIds = cmd.getVmIds();
+        Long clusterId = cmd.getId();
+
+        if (vmIds == null || vmIds.isEmpty() || clusterId == null) {
+            throw new InvalidParameterValueException("Invalid parameters specified");
```

Comment:
could be helpful to specify what parameter wasn't passed, i.e., vmIDs or clusterID